### DISCUSSION TITLE
How can I use KateX in mkdocs-material?

### DISCUSSION THREAD

**Discussion by nmnmcc** [squidfunk/mkdocs-material#4052] [Q&A] [answered]
I find that MathJax is much slower than KateX, and I want to choose what I use on my website.

I have tried to import KateX.min.js in mkdocs.yml, but it did not work.

**Comment by squidfunk** [comment, marked as answer]
Please see:
https://facelessuser.github.io/pymdown-extensions/extensions/arithmatex/#loading-katex
